app/styles/app/modules/request/form.scss

```diff
@@ -0,0 +1,93 @@
+.request-configs-form {
+  .config-type {
+    position: absolute;
+    top: 1px;
+    right: 0;
+    font-size: 10px;
+    z-index: 2;
+
+    * {
+      display: inline-block;
+      padding: 0 8px;
+      border: 1px solid transparent;
+      border-radius: 4px;
+    }
+
+    &.javascript .javascript,
+    &.yaml .yaml {
+      color: white;
+      background-color: $oxide-blue;
+      border-color: $oxide-blue;
+    }
+  }
+
+  .travis-form__field-component {
+    box-shadow: inset 0 0 1px 1px $dry-cement !important;
+  }
+
+  .CodeMirror {
+    height: 350px !important;
+    margin: -4px;
+    padding: 8px 12px 8px 4px;
+    width: calc(100% + 8px);
+    border-radius: 10px;
+    border: 4px solid transparent;
+    box-shadow: inset 0 0 1px 1px $dry-cement;
+    transition: border-color 0.2s ease-in;
+    color: $asphalt-grey;
+    line-height: 150%;
```

Comment:
This is minor/optional but for line-height a unitless value is usually preferred. In this case, it would be like `line-height: 1.5;`

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#%3Cnumber%3E

Reply:
interesting! i'll change this.

Reply:
addressed in https://github.com/travis-ci/travis-web/commit/18b6739765aeb142ff5a8cd69cfd652af9d6a8bb

Reply:
Looks good :+1: Thanks!